pkg/specgen/container_validate.go

```diff
@@ -13,6 +13,8 @@ var (
 	ErrInvalidSpecConfig = errors.New("invalid configuration")
 	// SystemDValues describes the only values that SystemD can be
 	SystemDValues = []string{"true", "false", "always"}
+	// SdNotifyModeValues describes the only values that SdNotifyMode can be
+	SdNotifyModeValues = []string{"container", "conmon-only", "ignore"}
```

Comment:
Note: this pkg doesn't have access to libpod/define - so I can't use the magic string replacements

Reply:
I think importing libpod/define here should be OK - we don't want to drag in libpod itself, but libpod/define is specifically structured to be just constants/struct definitions and very few external dependencies.